.github/workflows/release-please.yml

```diff
@@ -13,9 +13,21 @@ jobs:
       fail-fast: false
     steps:
       - uses: GoogleCloudPlatform/release-please-action@v2
+        id: release
```

Comment:
I think this should be ok as I'm assuming step ids are not shared across matrix job.
If this doesn't work we might need to do `id: release-{{ matrix-package }}`